packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1065,9 +1065,10 @@ export class AccountHandler {
     await this.customs.check(request, email, 'accountStatusCheck');
 
     try {
-      const exist = await this.db.accountExists(email);
+      const account = await this.db.accountRecord(email);
       return {
-        exists: exist,
+        exists: !!account,
```

Comment:
I think this can be replaced with `exists: true`? If no account is found, the new method would throw.